include/Color.hpp

```diff
@@ -208,15 +206,14 @@ class Color : public ::Color {
      */
     [[nodiscard]] Color Alpha(float alpha) const { return ::ColorAlpha(*this, alpha); }
 
-    Color Lerp(::Color color2, float factor) {
-        return ::ColorLerp(*this, color2, factor);
-    }
+    Color Lerp(::Color color2, float factor) { return ::ColorLerp(*this, color2, factor); }
 
     /**
      * Returns src alpha-blended into dst color with tint
      */
     [[nodiscard]] Color AlphaBlend(::Color dst, ::Color tint) const { return ::ColorAlphaBlend(dst, *this, tint); }
 
+    /* I think it deprecated
```

Comment:
Feel free to remove the old functions, I agree we shouldn't need these with the const expressions. May need to update some oft he examples though.